node/core/approval-voting/src/lib.rs

```diff
@@ -336,6 +346,16 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			unapproved_candidates_in_unfinalized_chain: prometheus::register(
+				prometheus::GaugeVec::new(
+					prometheus::Opts::new(
+						"polkadot_parachain_approval_unapproved_candidates_in_unfinalized_chain",
+						"Number of unapproved candidates in unfinalized chain",
+					),
+					&["count", "depth"]
```

Comment:
Ah, this is not what I meant. There should be one `label`, and it's value can be `count` or `depth`. I think it's better to have 2 separate Gauge in this case, one measuring depth of first unapproved and the total count of unapproved candidates.